js/ui/data_grid/ui.data_grid.base.js

```diff
@@ -141,6 +141,11 @@ var DataGrid = Widget.inherit({
                 },
                 options: {
                     grouping: {
+                        /**
+                         * @name dxDataGridOptions_grouping_expandMode
```

Comment:
Use "." instead of "_" 
I'm not sure whether we really need @publicName 
Let's ask @lukyanovas

Reply:
@TikhonovaTatyana, that's right